Test/Test.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
This project is a duplicate from  Group-11-Orleans/Test/DataModelsTest/DataModelsTest.csproj right? so we only need the other one